src/webportal/src/app/user/fabric/userView/index.jsx

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import React, {useState, useEffect, useMemo, useRef} from 'react';
+
+import {Fabric, Stack, initializeIcons} from 'office-ui-fabric-react';
+import {debounce} from 'lodash';
+
+import {MaskSpinnerLoading} from '../../../components/loading';
+import MessageBox from '../components/MessageBox';
+
+import Context from './Context';
+import TopBar from './TopBar';
+import Table from './Table';
+import Ordering from './Ordering';
+import Filter from './Filter';
+import Pagination from './Pagination';
+import Paginator from './Paginator';
+import {getAllUsersRequest, removeUserRequest, updateUserVcRequest, updateUserAccountRequest, updateUserGithubPATRequest} from '../conn';
+
+require('bootstrap/js/modal.js');
```

Comment:
Please use fabric's modal if possilble

Reply:
Changed to farbic's modal.
But for the inner html of modal, will update it to react component when designer give the new design.

Reply:
Fixed.